Gaijinizer patch/CommonEvents_561.txt

```diff
@@ -4,77 +4,97 @@
 
 コウビ！コウビッ！♡
 > CONTEXT: CommonEvents_561/13/Dialogue/enemy16
-Kahahahahahaha Ha!!Moth. Kobe!Koubi!Moth#MTLed
+Kyahahahahahahaha!!♡
+
+Mate! Mate!♡#Strate
 > END STRING
 
 > BEGIN STRING
 ワカッタ
 
 コンカイ　ハ　ヤメトク
 > CONTEXT: CommonEvents_561/25/Dialogue/enemy16
-Wacutta. Conkai Ha Yametoku#MTLed
+I get it.
+
+I stop this time.#Strate
 > END STRING
 
 > BEGIN STRING
 ヤッパリ　ニガサナイ…♡
 
 イッショニ　コウビシヨ…♡
 > CONTEXT: CommonEvents_561/32/Dialogue/enemy16
-Yappari Nigasanai...♡. together ビ シ...♡#MTLed
+I still not let go...♡.
+
+Mate together...♡#Strate
 > END STRING
 
 > BEGIN STRING
 ………？
 
 \\N[1]　ワタシガ　ミノガス　オモウノカ？
 > CONTEXT: CommonEvents_561/39/Dialogue/enemy16
-.........? \\N[1] 'I' Gasonomics Omounoka?#MTLed
+.........?
+
+\\N[1] thinks I gonna let him off?#Strate
```

Comment:
Same as before, might be more natural with "you" instead of protag's name and third person.